src/data_management/scenarios/Scenarios.js

```diff
@@ -114,27 +114,32 @@ class Scenarios extends Component {
                     <div className={scenartioStyle.Search}>
                         <SearchBox />
                     </div>
-                    <div className={scenartioStyle.Table}>
-                        {scenarioTableHeader()}
-                        <Scrollbars
-                            autoHeight
-                            autoHeightMin={500}
-                            autoHeightMax={500}
-                            style={{ width: "100%" }}
-                        >
-                            {scenarioTableBody(scenarios)}
-                            <div
-                                style={{
-                                    position: "absolute",
-                                    top: "45%",
-                                    left: "45%",
-                                    visibility: this.state.isFetching ? "visible" : "hidden"
-                                }}
+                    <Scrollbars
```

Comment:
@hoanphungt 
Hi Hoan overall looks good.
Just wondering why this scrollbar element is here.
It seems to not create a scrolling effect.
kr Tom